Assets/Project/Scripts/Common/Components/UIs/MessageDialog.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using Treevel.Common.Entities;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Treevel.Common.Components.UIs
+{
+    public class MessageDialog : MonoBehaviour
+    {
+        /// <summary>
+        /// ダイアログのタイプ
+        /// </summary>
+        public enum EDialogType {
+            Ok_Cancel, // 確定、キャンセルボタン付き
+            Ok, 　 // 確定ボタン付き
+        }
+
+        [SerializeField] private RectTransform _OkCancelTypeBtnPos;
```

Comment:
長いけど、どのボタンの pos か分からない。どの pos か分からないことよりも長いことを懸念するならやらなくていい
```suggestion
        [SerializeField] private RectTransform _OkCancelTypeOkBtnPos;
```

Reply:
8ddf0a2